Stylesheet/Components/textbox.css

```diff
@@ -0,0 +1,137 @@
+/* This is textbox file */
+
+input {
+    outline: none;
+    border: none;
+    display:block;
+  }
+
+label {
+  display: block;
+  font-size: 12px;
+  color: var(--st-bordergrey);
+} 
+  
+.basic-textfield-outlined {
+    outline: 2px solid transparent;
+    border: 1px solid var(--st-bordergrey);
+    border-radius: 0.3rem;
+    padding: 0.5rem;
+}
+
+.error-outlined {
+    border: 1px solid var(--st-red);
+}
+
+.basic-textfield-filled {
+    border: none;
+    padding: 0.5rem;
+    border-bottom: 1px solid var(--st-bordergrey);
+    border-top-left-radius: 0.3rem;
+    border-top-right-radius: 0.3rem;
+    background-color: var(--st-lightgrey);
+}
+
+.error-filled {
+    border-bottom: 1px solid var(--st-red);
+}
+
+.basic-textfield-standard {
+    border: none;
+    padding: 0.5rem;
+    border-bottom: 1px solid var(--st-bordergrey);
+}
+
+.error-standard {
+    border-bottom: 1px solid var(--st-red);
+}
+
+.help-block {
+    position: relative;
+    display: inline-block;
+}
+
+.helper-text-block {
+    position: absolute;
+    color: var(--st-bordergrey);
+    display: block;
+}
+
+.error-text {
+    color: var(--st-red);
+}
+
+.textfield-label {
+    display: inline-block;
+    padding: 3px;
+}
+
+.filled-color {
+    background-color: inherit;
+}
+
+.label-error {
+    padding-top: 6px;
+}
+
+.basic-textfield-outlined:hover {
+    outline: transparent;
+    border: 1px solid var(--st-darkgrey);
+}
+
+.error-outlined:hover {
+    border: 1px solid var(--st-darkred);
+}
+
+.basic-textfield-filled:hover {
+    outline: transparent;
```

Comment:
There are some properties that are repeating, you can look into that and reuse it